pyproject.toml

```diff
@@ -37,18 +37,14 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.10, <3.14"
-pygments = "^2"
-chardet = "^5"
-rich = ">=9, <14"
-gitpython = "^3"
+python = ">=3.11, <3.14"
+
 pyyaml = "^6.0.2"
 python-dotenv = "^1.0.1"
-pytest-mock = "^3.14.0"
 pydantic = "^2.10.2"
 pyjwt = "^2.9.0"
 requests-mock = "^1.12.1"
```

Comment:
`requests-mock` should have been removed from the main dependencies when it was moved to dev dependencies.